Defense evasion/base64-encoded-powershell-commands.md

```diff
@@ -0,0 +1,72 @@
+# Detect Base64-encoded PowerShell process and network creation commands
+
+This query was originally published in the threat analytics report, *Hunting for PowerShell Empire*
+
+[PowerShell Empire](https://www.powershellempire.com/) is a modular toolkit used both by penetration testers and malicious actors. It offers a wide range of attack techniques, and has been observed in numerous attacks.
+
+The following query detects Base64-encoded PowerShell commands that are either process creation or network events. This can identify common techniques, such as [Kerberoasting](https://docs.microsoft.com/azure-advanced-threat-protection/atp-reconnaissance-alerts#security-principal-reconnaissance-ldap-external-id-2038).
```

Comment:
@martyav @endisphotic, should we point this to:

https://attack.mitre.org/techniques/T1558/003/

Reply:
I updated this with the latest commit